### PR TITLE
Status E-Mail and Quality Scoring Update

### PR DESCRIPTION
**Pull request by bjbhatt**
@saracope - This pull request has following changes

1. Added Status Mailing functionality when Harvester run is completed.
2. Updated Quality Points Scoring Logic

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,9 +10,14 @@
...
-    "test": "./node_modules/.bin/nyc mocha --reporter xunit tests",
+    "test": "mocha --reporter xunit tests",
+    "test-coverage": "./node_modules/.bin/nyc npm test",
     "lint": "./node_modules/.bin/eslint .",
-    "lint-fix": "./node_modules/.bin/eslint --fix ."
+    "lint-fix": "./node_modules/.bin/eslint --fix .",
+    "postinstall": "opencollective-postinstall"
+  },
+  "collective": {
+    "url": "https://opencollective.com/webpack"
```
Comment (saracope) [inline package.json:20]: Do these Open Collective things still need to happen since we're disabling Open Collective in the manifest?

Comment (bjbhattGSA) [reply package.json:20]: It is not going to hurt, and all it does display a message when you do npm install "...please donate..."

**Comment by bjbhatt** [comment]
@saracope  This is the pull request for Harvester which include new Quality Point Scoring and also Summary E-Mail.  Sending summary E-Mail and all related configuration is configured through environment.  Until we figure out EMAIL Server related configuration from GSA, when can set the configuration to NOT send E-Mails out.

**Comment by bjbhatt** [comment]
[Harvester Run Sample E-Mail.pdf](https://github.com/GSA/code-gov-harvester/files/3182551/Harvester.Run.Sample.E-Mail.pdf)

@saracope Harvester Run Sample E-Mail Attached

**Pull request merged**